etc/geoserver/workspaces/MobiData-BW/ipl-db/sharing_stations_bicycle/featuretype.xml

```diff
@@ -100,6 +100,15 @@ WHERE st.num_bicycles_available &gt;= 0
         <de>Name des Fahrzeugs</de>
       </description>
     </attribute>
+    <attribute>
+      <name>capacity</name>
+      <minOccurs>0</minOccurs>
+      <maxOccurs>0</maxOccurs>
+      <binding>java.lang.Double</binding>
```

Comment:
I'd expect the capacity to be `java.lang.Integer` rather than `java.lang.Double`, no?

Reply:
@hbruch: Have you checked yet the definition of the attribute as `double precision` in the database?

Reply:
> @hbruch: Have you checked yet the definition of the attribute as `double precision` in the database?

Reminder.